nebula/tests/transcript/test_queue_worker.py

```
@@ -110,3 +110,197 @@ async def fake_light_phase(job_id, req, transcription, video_path, audio_path, u
+    await cancel_job(job_id)
+
+    # Mock the heavy operations to never actually run
+    async def fake_download(url, path):
+        pass
+
+    monkeypatch.setattr("nebula.transcript.queue_worker.download_video", fake_download)
+
+    # Try to run the heavy pipeline
+    try:
+        await qw._heavy_pipeline(job_id, req)
+        assert False, "Should have raised CancelledError"
```

Ruff (0.14.6)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)